src/openrct2/drawing/ImageIndexType.h

```diff
@@ -0,0 +1,17 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2024 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <cstdint>
+#include <limits>
+
+using ImageIndex = uint32_t;
+
+constexpr ImageIndex kSpriteIdNull = std::numeric_limits<ImageIndex>::max();
```

Comment:
Files should end with a newline.

```suggestion
constexpr ImageIndex kSpriteIdNull = std::numeric_limits<ImageIndex>::max();

```